.azure-pipelines.yml

```diff
@@ -29,9 +29,9 @@ stages:
 - stage: build_test
   displayName: Run
   jobs:
-  - template: .azure-pipelines/docker.yml
-  - template: .azure-pipelines/detekt.yml
-  - template: .azure-pipelines/linux-test.yml
-  - template: .azure-pipelines/linux-analyzer-test.yml
-  - template: .azure-pipelines/windows-test.yml
-  - template: .azure-pipelines/windows-analyzer-test.yml
+  - template: .azure-pipelines/Docker.yml
```

Comment:
Is there really any value in this alignment? Because the root file *must* be called `.azure-pipelines.yml`, so it uses dashes instead of CamelCase, and we should probably keep doing the same here.

Reply:
Yes, job names are aligned with filenames, so we don't use different spellings for the same thing. The root file could actually be called anything we like, because you can select a file when creating the pipeline.

Reply:
I'm still not convinced, as the default name is `azure-pipelines.yml` (without the leading dot, though), and we use dashes for all other `.yml` files in this repo, but I'll also not block this change.